app/views/editor/application/_navigation.html.haml

```diff
@@ -0,0 +1,14 @@
+.page-header
+  %nav.navbar.navbar-default
+    .container-fluid
+      .navbar-header
+        .navbar-brand "#{current_site.name}"
+
```

Comment:
Is it intentional what the switching link to other site and the logo image has gone?